guide/src/python_from_rust.md

```diff
@@ -139,7 +139,7 @@ fn main() -> PyResult<()> {
 
 ### Want to run just an expression? Then use `eval`.
 
-[`Python::eval`]({{#PYO3_DOCS_URL}}/pyo3/struct.Python.html#method.eval) is
```

Comment:
preview: https://deploy-preview-3481--pyo3.netlify.app/main/python_from_rust#want-to-run-just-an-expression-then-use-eval